xbmc/filesystem/test/TestZipFile.cpp

```diff
@@ -16,6 +16,7 @@
 #include "test/TestUtils.h"
 #include "utils/StringUtils.h"
 #include "utils/URIUtils.h"
+#include "filesystem/ZipFile.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers